README.md

```diff
@@ -66,6 +66,14 @@ You will see something like in the following _maxGraph panning_ demo:
 For more details, have a look at the [storybook stories](packages/html/stories).
 
 
+## Typescript
```

Comment:
It'd be better to change this to `TypeScript`.

Reply:
done in 1f22d59